conbench/tests/api/test_commits.py

```diff
@@ -40,3 +40,27 @@ def test_commit_list(self, client):
         commit = self._create()
         response = client.get("/api/commits/")
         self.assert_200_ok(response, contains=_expected_entity(commit))
+
+    def test_commit_list_filter_by_sha(self, client):
+        sha = _fixtures.CHILD
+        self.authenticate(client)
+        commit = self._create()
+        response = client.get(f"/api/commits/?commit={sha}")
+        self.assert_200_ok(response, contains=_expected_entity(commit))
+
+    def test_commit_list_filter_by_multiple_sha(self, client):
+        sha1 = _fixtures.CHILD
+        sha2 = _fixtures.PARENT
+        self.authenticate(client)
+        _fixtures.benchmark_result(sha=sha1)
+        benchmark_result1 = _fixtures.benchmark_result()
+        _fixtures.benchmark_result(sha=sha2)
+        benchmark_result2 = _fixtures.benchmark_result()
+        response = client.get(f"/api/commits/?commit={sha1},{sha2}")
```

Comment:
`benchmark_result1` and `benchmark_result2` -> `benchmark_result_1` and `benchmark_result_2`